model_compression_toolkit/target_platform_capabilities/tpc_models/imx500_tpc/v1/tp_model.py

```diff
@@ -102,7 +102,7 @@ def get_op_quantization_configs() -> Tuple[OpQuantizationConfig, List[OpQuantiza
         signedness=Signedness.AUTO)
 
     # We define an 8-bit config for linear operations quantization, that include a kernel and bias attributes.
-    linear_eight_bits = tp.OpQuantizationConfig(
+    linear_eight_bits = model_compression_toolkit.target_platform_capabilities.schema.v1.OpQuantizationConfig(
```

Comment:
maybe import the path as "schema_v1" (or create an alias at the beginning of the file) instead of repeating it everywhere?
or do we want it to be explicit in the TPC model description? @haihabi 
even if we want it to, eventually this file will be a JSON and not written as code using imports, so these fields in the JSON won't include the entire schema path anyway...

Reply:
But the JSON file should have the schema version from the constant in the schema main class.

Reply:
> But the JSON file should have the schema version from the constant in the schema main class.

So you agree? the path here can be shortened with an alias to improve file readability?

Reply:
Yes